<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI</class>
 <widget class="QMainWindow" name="GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>807</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="OnButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QFrame" name="video_frame_1">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>40</y>
      <width>251</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="video_frame_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>220</y>
      <width>251</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="video_frame_3">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>400</y>
      <width>251</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="video_label">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Видео с камер</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>16</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>16</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>16</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>220</y>
      <width>16</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>471</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>471</width>
      <height>71</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="Actual_tool_pose_label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>111</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Actual tool pose</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ManualCartButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Manual Cart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ManualJointButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>70</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Manual Joint</string>
    </property>
   </widget>
   <widget class="QLabel" name="For_manual_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>For Manual</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>16</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_6">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>16</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
